Gemfile

```diff
@@ -76,7 +76,7 @@ gem 'formtastic', '~> 4.0'
 gem 'htmlentities', '~>4.3', '>= 4.3.4'
 # TODO: Not actively maintained https://github.com/activeadmin/inherited_resources#notice replace with respond_with and fix things the rails way
 gem 'inherited_resources', '~> 1.12.0'
-gem 'json', '~> 2.3.0'
+gem 'json', '~> 2'
```

Comment:
If your intention is to limit version to `<3`, then maybe use `~> 2.3` so that we don't go back with the version. Although unlikely to do either way.

Reply:
Well, in fact I only updated this gem because I have so many bundles locally I was getting an odd error about `json 2.3.0` conflicting with another repo where I use the same ruby version but `json 2.6.3` . This is a problem in my environment, not porta, but updating porta to `json 2.6.3` was the quickest way to fix it, so I thought: why not? 😅. Anyway you're right. I'll set the version to 2.6.3 explicitly.

Reply:
`~> 2.3` will allow for `2.6.3`. No need to hardcode it. Otherwise next one looking at it would be wondering, why on earth did we hardcode to `2.6.3`? Maybe later versions have a bug? Maybe I shouldn't touch it...

Reply:
Done